frontend/src/e2e-test/pages/admin/application-workbench-page.ts

```diff
@@ -178,6 +185,19 @@ export class ApplicationWorkbenchPage {
     await this.waitUntilLoaded()
   }
 
+  async assertApplicationStatusTextMatches(
+    index: number,
+    matchingText: string
+  ) {
+    await waitUntilTrue(async () => {
+      await this.#applicationPlacementProposalStatusIndicator.hover()
+      const tooltip = await this.#applicationPlacementProposalStatusTooltip
+        .innerText
+      const match = tooltip.match(matchingText)
+      return match ? match.length > 0 : false
+    })
+  }
+
```

Comment:
I think using `waitUntilTrue` is not very useful here. Playwright already waits that the status indicator is in the DOM for `.hover()` to succeed. If the tooltip text is incorrect and `waitUntilTrue` retries, hovering that element again doesn't change anything as the tooltip is already visible.